src/models/user

```diff
@@ -0,0 +1,218 @@
+const mongoose = require('mongoose')
+
+const cities = ['Adana', 'Kocaeli', 'Adiyaman', 'Konya'
```

Comment:
for const variables lets create a variables and make sure its captial case, save it in another file and import it,
you can use the following pattern for it: https://stackoverflow.com/questions/8595509/how-do-you-share-constants-in-nodejs-modules